docker/data-science/notebooks/aips.py

```diff
@@ -23,44 +23,44 @@
 WEBSERVER_URL = f'http://{AIPS_WEBSERVER_HOST}:{AIPS_WEBSERVER_PORT}'
 
 def healthcheck():
-  status_url = f'{SOLR_URL}/admin/zookeeper/status'
+    status_url = f'{SOLR_URL}/admin/zookeeper/status'
 
-  try:
-    if (get_engine().health_check()):
-      print ("Solr is up and responding.")
-      print ("Zookeeper is up and responding.\n")
-      print ("All Systems are ready. Happy Searching!")
-  except:
-      print ("Error! One or more containers are not responding.\nPlease follow the instructions in Appendix A.")
+    try:
+        if (get_engine().health_check()):
```

Comment:
do we need to have `if` here if the function throws an exception anyway?

Reply:
There are instances where health check doesn't throw an exception but the healthcheck fails. I'm seeing now that the else case doesn't return the failure message. Will fix that